### DISCUSSION TITLE
大佬们，这是啥问题

### DISCUSSION THREAD

**Discussion by yosemite118** [jeessy2/ddns-go#1147] [Q&A]
NAS停电后再重启就变这样了，其它服务都正常，由于太久没研究NAS、Docker，无从下手，前来寻求大佬帮助

![1717726014887](https://github.com/jeessy2/ddns-go/assets/62743984/50f31cab-a937-44d8-bb0c-71982703fbf9)

**Comment by WaterLemons2k** [comment]
换个接口

**Comment by jeessy2** [comment]
参考#1047

**Reply by yosemite118** [reply]
谢谢大佬，可以了
